Counterexamples/AharoniKorman.lean

```diff
@@ -8,7 +8,7 @@ import Mathlib.Algebra.Order.Field.Basic
 import Mathlib.Algebra.Order.Field.Rat
 import Mathlib.Algebra.Order.Star.Basic
 import Mathlib.Data.Nat.Cast.Order.Ring
-import Mathlib.Order.Chain
+import Mathlib.Order.CompleteLattice.Chain
```

Comment:
I'm actually surprised this is needed - I don't think I used anything about maximality in this file. Does it work to import Order/Chain and CompleteLattice separately?

Reply:
I don't think you were using anything about star-ordered rings either 😉